windi.config.ts

```diff
@@ -56,6 +60,9 @@ export default defineConfig({
       ),
     extend: {
       screens: {
```

Comment:
these screens are already provided by windicss—extend should prevent our theme from overwriting them, was that not the case?